### DISCUSSION TITLE
Error: 500 Line too long (see RFC5321 4.5.3.1.6)

### DISCUSSION THREAD

**Discussion by karlis-vagalis** [YoRyan/mailrise#80] [Q&A] [answered]
Hello,

thanks for publishing and working on such an amazing project!

I have already gotten Mailrise working and passing notifications correctly, but have encountered, that one of the clients (my network router) which tries to connect to Mailrise unfortunately does not comply to the RFC5321 4.5.3.1.6 standard. Is there any built in workaround to extend the allowed message length in Mailrise?

The error I get is `<< b'500 Line too long (see RFC5321 4.5.3.1.6)'`

Would highly appreciate any help or suggestions!

**Comment by YoRyan** [comment]
This should work now with f9798231eb44b6fc9f4732e7fd8642d478c65dc1. Cheers!

**Reply by karlis-vagalis** [reply]
Thanks for the quick fix!

Thus far I was using `:stable` tag and everything was working.
Now, to get the patch when I switched to the `:latest` tag, the mailrise unfortunately does not work anymore. The only log output from container I get is:
```
[<timestamp>] INFO:mailrise.skeleton:Loaded configuration with 1 recipient(s)
[<timestamp>] DEBUG:mailrise.skeleton:Arguments for aiosmtpd: authenticator=Basic(1), auth_required=True, auth_require_tls=False, tls_context=None, ssl_context=None, require_starttls=False
[<timestamp>] DEBUG:asyncio:Using selector: EpollSelector
```
and no further messages from any of the submitted requests.


To debug, I also tried:

1. While using `:latest`, I thought that maybe default config of mailrise has changed, so I tried to set
```
tls:
    mode: off
```
which threw an error:
```
Traceback (most recent call last):

  File "/home/mailrise/.local/bin/mailrise", line 8, in <module>

    sys.exit(run())

             ^^^^^

  File "/home/mailrise/.local/lib/python3.11/site-packages/mailrise/skeleton.py", line 169, in run

    main(sys.argv[1:])

  File "/home/mailrise/.local/lib/python3.11/site-packages/mailrise/skeleton.py", line 107, in main

    config = load_config(_logger, pargs.config)

             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/mailrise/.local/lib/python3.11/site-packages/mailrise/config.py", line 120, in load_config

    yml_tls_mode = yml_tls.get('mode', 'off').upper()

                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

AttributeError: 'bool' object has no attribute 'upper'
```

Is there something I should consider to fix this switch from `stable` to `latest`? Thanks in advance!

**Reply by YoRyan** [reply, marked as answer]
Right, the new build was actually completely broken. That's just what I get for not testing the code before pushing it out.

As for the `mode: off` error, that was yet another, unrelated one that's been in there for awhile. Nice catch.

All should be fine now.

**Reply by karlis-vagalis** [reply]
Tried the new `latest` build, works perfect! Also the the line lenght fix did it for my SMTP client! Thanks you so much!
